src/components/Game.js

```diff
@@ -412,6 +412,7 @@ function Game() {
             <div style={{ position: "relative" }}>
               <img
                 // height="13px"
+                className="img-responsive"
```

Comment:
will this class name can be applied for all images?

Reply:
not used Anywhere removed it